index.bs

```diff
@@ -1077,15 +1101,23 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
           </ol>
         </li>
         <li>
-          Let <var>p</var> be a new promise.
+          Let <var>p</var> be a new Promise.
         </li>
         <li>
-          <a>In parallel</a>, run the <a>Create a
-          MediaCapabilitiesDecodingInfo</a> algorithm with
-          <var>configuration</var> and resolve <var>p</var> with its result.
+          Return <var>p</var>.
```

Comment:
If I read [Infra](https://infra.spec.whatwg.org/#algorithm-control-flow) correctly, a "return" statement terminates the algorithm. This step and the following one should rather be swapped.

Reply:
Done